x-pack/test/security_solution_endpoint/page_objects/endpoint_page.ts

```diff
@@ -33,6 +33,16 @@ export function EndpointPageProvider({ getService, getPageObjects }: FtrProvider
       });
     },
 
+    async waitForTableToNotHaveData(dataTestSubj: string) {
```

Comment:
This is a nice one and I can see us moving this to a page_utils page object in the future as a common way to wait for a table to empty out 👍

Reply:
Just a small comment: we should add JSDocs to this method. I think normally, the page objects can be reused by other tests (even outside of Endpoint) to integrate/interact with endpoint functionality